source/support.tex

```diff
@@ -5365,10 +5469,24 @@
     constexpr void* address() const noexcept;
   private:
     coroutine_handle(@\unspec@);
+    void* ptr;  // \expos
   };
 }
 \end{codeblock}
 
+\rSec4[coroutine.handle.noop.conv]{Conversion}
+
+\indexlibrarymember{operator bool}{coroutine_handle<noop_coroutine_promise>}%
```

Comment:
```suggestion
\indexlibrarymember{operator coroutine_handle<>}{coroutine_handle<noop_coroutine_promise>}%
```

Reply:
All excellent catches so far - I should really be more careful when I cut and paste!